src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    count++;
+
+    return (y) => {
+      if (count < 3 || count % 2 !== 0) {
+        return x + y;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
```suggestion
  return (x) => {
    count++;

    return (y) => {
      if (count < 3 || count % 2 !== 0) {
        return x + y;
      }

      return 'Bzzz... Error!';
      
       return (x) => (y) => {
    counter++;

    return counter > 3 && counter % 2 === 0
      ? 'Bzzz... Error!'
      : x + y;
  };
```

It's just a suggestion, I've also did it the same way as you, but after I've finished the tests I tried to shorten the code.
And since the ternary operators were question for me in the TC, I giving you the shorter just as an example on how to shorten the code.